swagger/src/main/scala/org/http4s/rho/swagger/TypeBuilder.scala

```diff
@@ -80,6 +80,35 @@ object TypeBuilder {
 
           models ++ generics ++ children
 
+        case tpe@TypeRef(_, sym: Symbol, tpeArgs: List[Type]) if isSumType(sym) =>
+          // TODO promote methods on sealed trait from children to model
+          modelToSwagger(tpe, sfs).map({ m =>
```

Comment:
Sure. I've developed the habit of putting both everywhere, call it defensive syntax.